src/GHC/Util.hs

```diff
@@ -17,10 +17,12 @@ module GHC.Util (
   , parsePragmasIntoDynFlags, parseFileGhcLib
   ) where
 
+import GHC.Util.Brackets
```

Comment:
Why touch this module but not change its API?

Reply:
An error. Vestige of an intermediate approach. Will remove.